client/vscode/src/sampleCommandActivation.ts

```diff
@@ -14,3 +18,59 @@ export function logCommand(languageClient: LanguageClient) {
         languageClient.info(`Client: The log command has been executed`)
     }
 }
+
+export function qNetCommand(languageClient: LanguageClient) {
+    return async () => {
+        const request = {
+            command: 'aws/qNetTransform/startTransform',
+            SolutionRootPath: 'D:\\TestProjects-master\\TestProjects-master\\net40\\mvc\\MvcMusicStore.sln',
+            TargetFramework: 'net8.0',
+            SourceCodeFilePaths: [
```

Comment:
Do these have to be relative to a `workspaceFolder` (https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_workspaceFolders)

I'm not sure the server will be able to get them otherwise? Or at least, the server shouldn't access the filesystem outside of the workspace to prevent unintended or unauthorized access to the user's filesystem.

Reply:
is lsp server process act as subprocess in the visual studio? the source code files are subset of workspace folder but we also need "references", its not necessary to be in the same folder (customer can chose the nuget source from anyplace on the local). Visual studio would definitely have the access. Is LSP process have same permission as visual studio?

Reply:
Yes, the LSP process has the same permissions. I just wanted to make sure we limit filesystem access to no more than we need for the use case.